samples/caf_sensor_manager/README.rst

```diff
@@ -0,0 +1,76 @@
+.. _caf_sensor_manager_sample:
+
+CAF Sensor Manager
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The CAF Sensor Manager sample demonstrates the functionality of the :ref:`caf_sensor_manager`.
+It shows how to use the Sensor Manager to sample data, aggregate data and then receive them in packages.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840
+
+Overview
+********
+
+The sample:
+
+1. Provides simulated input data using sensor_sim and :ref:`caf_sensor_manager`.
+#. Aggregates data using :ref:`caf_sensor_data_aggregator`.
+#. When aggregator buffer is full sends data to workload_sim module.
+#. Simulates work using workload_sim.
+
+Configuration
+*************
+
+|config|
```

Comment:
Remove these lines unless there's optional configuraton that can be done by the user.